src/env.cc

```diff
@@ -512,11 +522,16 @@ void TrackingTraceStateObserver::UpdateTraceCategoryState() {
 
 void Environment::AssignToContext(Local<v8::Context> context,
                                   const ContextInfo& info) {
+  AssignToContext(context, nullptr, info);
```

Comment:
Do we need this alias? I think technically we can just use `env->principal_realm()` for all existing calls to `AssignToContext()` (including the one in vm contexts)?

Reply:
Yeah, this method is only used in a few places. Updating their call sites instead of creating new aliases sounds more reasonable. Thanks for pointing this out. Updated.